src/components/Popper/Popper.stories.tsx

```diff
@@ -4,29 +4,20 @@ import React, { useState } from 'react';
 import { Meta, StoryFn } from '@storybook/react';
 import { Popper } from './Popper';
 import { Button } from '@components/Button';
-import { PopperPlacement, PopperProps } from '@components/Popper/types';
+import { POPPER_STORY_ARGS, PopperPlacement, PopperProps } from '@components/Popper/types';
 import { Box } from '@components/Box';
 
 export default {
     title: 'Utilities/Popper',
     component: Popper,
     tags: ['autodocs'],
     args: {
-        placement: 'bottom-start',
+        placement: PopperPlacement.BottomStart,
```

Comment:
As Popper component isn't used that much (yet), I would go union type route instead of enum